Source/JSON.swift

```diff
@@ -409,4 +397,44 @@ extension JSON: CustomDebugStringConvertible {
     public var debugDescription: String {
         return PrettyJSONSerializer().serializeToString(json: self)
     }
+}
+
+public protocol JSONConvertible {
+    func makeJSON() -> JSON
+}
+
```

Comment:
Zewo usually uses `Representable` as the suffix for when you can represent a type with another type. Representables are usually a property as well.

```swift
public protocol JSONRepresentable {
    var json: JSON { get }
}
```

Would this still work? It wouldn't be a problem would it?